webknossos-datastore/app/com/scalableminds/webknossos/datastore/models/datasource/DataLayer.scala

```diff
@@ -148,6 +148,8 @@ trait DataLayerLike {
 
   def resolutions: List[Vec3Int]
 
+  lazy val sortedResolutions: List[Vec3Int] = resolutions.sortBy(_.maxDim)
```

Comment:
```suggestion
  lazy val sortedMags: List[Vec3Int] = resolutions.sortBy(_.maxDim)
```

I know, this is a bit of a mess. Unfortunately, we cannot rename `resolutions` itself in DataLayer, [as previously discussed](https://github.com/scalableminds/webknossos/pull/8111#discussion_r1801110648). However, I’d like to avoid the word "resolutions" propagating from there into the codebase. So I’d actually go with sortedMags here. What do you think?

Reply:
sure, of cause :+1: